entrypoint

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+set -e
+
+RUN_COMMAND="talisker.gunicorn.gevent webapp.app:create_app() --bind $1 --worker-class gevent --name talisker-`hostname`"
```

Comment:
just so that it's done and we don't forget it. Now we enable access-logs for gunicorn. Can you just replace with this line:
```bash
RUN_COMMAND="talisker.gunicorn.gevent webapp.app:create_app() --bind $1 --worker-class gevent --name talisker-`hostname` --access-logfile -"
```

(last paramater is the important one :smile: )